templates/author.twig

```diff
@@ -0,0 +1,72 @@
+{% extends "base.twig" %}
+
+{% block content %}
+    <header class="page-header container">
+        {% if ( author.avatar ) %}
+            <figure class="author-image" aria-label="{{ __( 'Picture of ' ~ author.name , 'planet4-master-theme' ) }}">
+                <img itemprop="image" class="author-pic"
+                     src="{{ fn('get_avatar_url', author.id, {'size' : 300}) | replace({'=s96-c':""}) }}"
+                     alt="{{ author.name }}">
+            </figure>
+        {% endif %}
+        <div class="author-details">
+            <h1 class="page-header-title">{{ author.name }}</h1>
+
+            {% if role %}
+                <h2>{{ role|raw }}</h2>
```

Comment:
this probably needs to be sanitized for security, see comment below,

@comzeradd do you have any guidance on this we could refer to?

Reply:
You can use `wp_kses_post`, as we do in the [original template](https://github.com/greenpeace/planet4-master-theme/blob/main/templates/author.twig#L15). That allows specific html tags only, that we define in [a filter](https://github.com/greenpeace/planet4-master-theme/blob/983692440e648f2e3e730d25345cec000143b3a5/src/MasterSite.php#L766-L873).